doc/parameters/param_list.md

```diff
@@ -155,8 +155,11 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
 | progress_checker_plugin | "progress_checker" | Check the progress of the robot |
+| progress_checker.plugin | "nav2_controller::SimpleProgressChecker" | Default plugin |
```

Comment:
Along side mentioning that this is default plugin class for progress_checker, can we mention that "progress_checker" in `progress_checker.plugin` parameter name is coming from the  `progress_checker_plugin` parameter? And that it should correspond to whatever plugin class name intended to be used.

Reply:
Ya I was wondering what the right convention is here. `<progress checker id>.plugin`?

Reply:
Yeah that would work - I think you could also just check what was done for the planner / controller plugins that were already updated and just mirror it

Reply:
Done. No one .plugin param is mentioned in this file yet.